js/index.js

```diff
@@ -0,0 +1,60 @@
+
+var user_selection = prompt("What Calculator mode do you want to use? Basic, Advanced,  or BMI?");
+
+// Prompt Menu
+
+if (user_selection == "basic"){
```

Comment:
This works, but only if the options are exactly typed. This is a bit awkward for the user and confusing because you ask them to type "Basic", but only accept "basic" with a lowercase b. I would probably make the option smaller, like (a) or (1). Try to keep this consideration of user experience in your mind in the future.